src/pages/stablecoins.tsx

```diff
@@ -33,241 +26,16 @@ import Translation from "../components/Translation"
-  box-shadow: 8px 8px 0px 0px ${(props) => props.theme.colors.gridBlue};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-left: 0rem;
-  }
-`
-
-const DaiSubtitle = styled.p`
-  font-size: 1.25rem;
-  line-height: 140%;
-  color: ${(props) => props.theme.colors.text200};
-`
-
-const H2 = styled.h2`
-  margin-top: 0rem;
-`
-
-const H3 = styled.h3`
-  font-size: 1.25rem;
-  font-style: normal;
-  font-weight: 700;
-  text-align: left;
-  margin-top: 0;
-  margin-bottom: 1rem;
-`
-/* const OptionContainer = styled.div`
-  display: flex;
-  justify-content: center;
-  padding: 0 2rem;
-  margin-bottom: 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-    width: 100%;
-  }
-` */
-
-const StyledCalloutBanner = styled(CalloutBanner)`
-  margin: 2rem 0 4rem;
-`
-
-const FullWidthContainer = styled(Page)`
-  padding: 0rem 2rem;
-  padding-bottom: 4rem;
-  margin-top: -2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: 0rem;
-    padding-left: 0rem;
-    padding-right: 0rem;
-    margin-top: -2rem;
-  }
-`
-
-const DaiH2 = styled.h2`
-  font-size: 2rem;
-  font-weight: 700;
-  margin-top: 0;
-  margin-bottom: 0.5rem;
-
-  a {
-    display: none;
-  }
-`
-
-const LeftColumn = styled.div`
-  margin-right: 2rem;
-  width: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: auto 0rem;
-  }
-`
-
-const StyledRightColumn = styled.div`
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  border-radius: 2px;
-  padding: 2rem;
-  width: 100%;
-  border: 1.5px solid ${(props) => props.theme.colors.text};
-  box-shadow: ${(props) => props.theme.colors.cardBoxShadow};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: auto 0rem;
-  }
-`
-const TokenCard = styled(HorizontalCard)`
-  min-width: 100%;
-  margin: 0.5rem 0rem;
-  border-radius: 0px;
-`
-
-const StyledLeftColumn = styled(LeftColumn)`
-  justify-content: center;
-  display: flex;
-  flex-direction: column;
-  width: 50%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    width: 100%;
-  }
-`
-
-const StyledCardGrid = styled(CardGrid)`
-  margin-bottom: 4rem;
-`
-
-const TableContent = styled(Content)`
-  overflow-x: scroll;
-`
-
-const ButtonColumn = styled.div`
-  display: flex;
-  flex-direction: column;
-`
-
-const Apy = styled.p`
-  font-size: 64px;
-  line-height: 100%;
-`
-
-const InfoIcon = styled(Icon)`
-  margin-left: 0.5rem;
-  fill: ${(props) => props.theme.colors.text};
-`
-
-const ButtonLinkWrap = styled(ButtonLink)`
-  white-space: break-spaces;
-`
-
-const ButtonLinkContainer = styled.div`
-  display: flex;
-  flex-direction: row;
-  flex-flow: wrap;
-  gap: 1em;
-`
-
-const PaddedContent = styled(Content)`
-  padding-top: 3rem;
-  padding-bottom: 3rem;
-`
+import {
+  Box,
+  Flex,
+  Heading,
+  Text,
+  useToken,
+  Img,
+  Icon,
+  Grid,
+} from "@chakra-ui/react"
```

Comment:
import order, this should be on top of the file, next to the other external lib imports